lib/info.js

```diff
@@ -330,7 +330,19 @@ const parseFormats = player_response => {
  * @param {Object} options
  * @returns {Promise<Object>}
  */
+const { version } = require('../package.json');
+let lastUpdateCheck = Date.now();
 exports.getInfo = async(id, options) => {
+  update = process.env.UPDATE;
+  miniget('https://api.github.com/repos/fent/node-ytdl-core/releases/latest', { headers: { 'User-Agent': 'a/b' } })
+    .text().then(response => {
+      if (update === 'always' || (update !== 'never' && Date.now() - lastUpdateCheck >= 43200000)) {
```

Comment:
would prefer if the number here was not precalculated so that someone looking at this can know how much time it is. `43200000` -> `1000 * 60 * 60 * 12`